tests/foreman/api/test_template.py

```diff
@@ -42,303 +44,232 @@
-        template.template_kind = None
-        template.template_kind_name = gen_string('alpha')
-        with self.assertRaises(HTTPError) as context:
-            template.create(create_missing=False)
-        self.assertEqual(context.exception.response.status_code, 422)
-        self.assertRegex(context.exception.response.text, "Could not find template_kind with name")
-
-    @tier1
-    def test_positive_update_name(self):
-        """Create configuration template providing the initial name,
-        then update its name to another valid name.
-
-        :id: 58ccc4ee-5faa-4fb2-bfd0-e19412e230dd
-
-        :expectedresults: Configuration Template is created, and its name can
-            be updated.
-
-        :CaseLevel: Component
-
-        :CaseImportance: Critical
-        """
-        c_temp = entities.ConfigTemplate().create()
-
-        for new_name in valid_data_list():
-            with self.subTest(new_name):
-                updated = entities.ConfigTemplate(id=c_temp.id, name=new_name).update(['name'])
-                self.assertEqual(new_name, updated.name)
-
-    @tier1
-    def test_positive_update_with_manager_role(self):
-        """Create template providing the initial name, then update its name
-        with manager user role.
-
-        :id: 0aed79f0-7c9a-4789-99ba-56f2db82f097
-
-        :expectedresults: Provisioning Template is created, and its name can
-            be updated.
-
-        :CaseImportance: Medium
-
-        :BZ: 1277308
-        """
-        user_login = gen_string('alpha')
-        user_password = gen_string('alpha')
-        new_name = gen_string('alpha')
-        org = entities.Organization().create()
-        loc = entities.Location().create()
-        template = entities.ProvisioningTemplate(organization=[org], location=[loc]).create()
-        # Create user with Manager role
-        role = entities.Role().search(query={'search': 'name="Manager"'})[0]
-        entities.User(
-            role=[role],
-            admin=False,
-            login=user_login,
-            password=user_password,
-            organization=[org],
-            location=[loc],
+        template = entities.ProvisioningTemplate(
+            name=name,
+            organization=[module_org],
+            location=[module_location],
+            snippet=False,
+            template_kind=template_kind,
         ).create()
-        # Update template name with that user
-        cfg = get_nailgun_config()
-        cfg.auth = (user_login, user_password)
+        assert template.name == name
+        assert len(template.organization) == 1, "Template should be assigned to a single org here"
+        assert template.organization[0].id == module_org.id
+        assert len(template.location) == 1, "Template should be assigned to a single location here"
+        assert template.location[0].id == module_location.id
+        assert template.snippet is False, "Template snippet attribute is True instead of False"
```

Comment:
```python
assert not template.snippet
```
?
Or `assert template.snippet == False`. Linter might complain about this one. But seems more natural, as we want to compare values, not if specific variable points to same object.

Reply:
If i recall correctly, my version was suggested by the linter :)

Reply:
Oh, ok. It's not a huge problem anyway.